package.json

```diff
@@ -43,8 +43,6 @@
     "ajv": "^6.0.0",
     "ansi-red": "^0.1.1",
     "appium-ci": "^0.3.0",
-    "appium-support": "^2.25.0",
```

Comment:
it might be handy to add a comment here about including depending packages

Reply:
Unfortunately JSON doesn't allow comments.